test/e2e/specs/site-editor/navigation-editor.spec.js

```diff
@@ -62,11 +62,9 @@ test.describe( 'Editing Navigation Menus', () => {
 			await expect( navBlockNode ).toBeVisible();
 
 			// The Navigation block description should contain the locked state information.
-			const navBlockNodeDescriptionId =
-				await navBlockNode.getAttribute( 'aria-describedby' );
-			await expect(
-				listView.locator( `id=${ navBlockNodeDescriptionId }` )
-			).toHaveText( /This block is locked./ );
+			await expect( navBlockNode ).toHaveAccessibleDescription(
+				/This block is locked./
+			);
```

Comment:
@afercia, I think you'll like the new a11y assertions - https://github.com/microsoft/playwright/releases/tag/v1.44.0

Reply:
Uhu, thanks for pointing me at it 🥳